portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,24 @@
+@import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Mono&family=IBM+Plex+Sans&display=swap');
+
+:root, body, html {
+  font-size: 16px;
+  font-family: 'IBM Plex Sans', sans-serif;
+}
+
+header, footer {
+  display: flex;
+  flex-flow: column nowrap;
+  align-items: center;
+}
+
 #content {
```

Comment:
Consider using classes instead of id's. Browsers assert that there's only one instance of each id on a page, which makes them very fast to query for but easy to misuse. Especially for generic container-like elements, it's preferred to use classes.

Reply:
But there is only one `#content` element per page, so isn't this in line with what I intended?

Reply:
Yeah, this one's up to you. It's best practice to not use _general_ id names (like "content", "nav", or "footer") since there are potentially multiple of these that could be added to the page in the future. Normally reserving them for id's like "cheesecake-recipe" since there's not a likely namespace collision. Again, it's fine here, but something to keep in mind when collaborating on larger projects.